### ISSUE TITLE
Travis build issue with swift 3 project

### ISSUE THREAD

**Issue by wstyres** [rs/SDWebImage#1817]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | iOS
 Platform Version        | 10.0
 SDWebImage Version      | 4.0
 Integration Method      | cocoapods
 Xcode Version           | Xcode 8.2
 Repro rate              | 100%

### Issue Description and Steps

When using SDWebImage installed with cocoapods and building with travis, travis fails everytime building any file with SDWebImage inside of it.

The error is as follows:
```
NewsStoryView.swift:24:9: error: value of type 'UIImageView' has no member 'sd_setShowActivityIndicatorView'
        imageView.sd_setShowActivityIndicatorView(true)
        ^~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<unknown>:0: note: did you mean 'setShowActivityIndicatorView'?
NewsStoryView.swift:25:9: error: value of type 'UIImageView' has no member 'sd_setIndicatorStyle'
        imageView.sd_setIndicatorStyle(.gray)
        ^~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~
SDWebImage.UIImageView:164:15: note: did you mean 'setIndicatorStyle'?
    open func setIndicatorStyle(_ style: UIActivityIndicatorViewStyle)
              ^
```

The build only fails on travis and works perfectly when building with Xcode and for Release because I followed #1764 with adding NS_REFINED_FOR_SWIFT in some of the source files and adding SDWebImage manually into my project libraries.

**Comment by skyline75489** [comment]
I don't quite understand the issue. Is #1764 making the travis build failing?

**Comment by wstyres** [comment]
Using some fixes mentioned in the comments of #1764 actually allows me to build and run the Xcode project at all.  Without them I can't use SDWebImage without getting an unrecognized selector error.  But with or without the fixes I added, the travis build will still fail

**Comment by wstyres** [comment]
Turns out travis-ci was using an outdated pod repo and was downloading version 3.8.6 as opposed to version 4.0.0. By adding `pod update repo` into my travis build file the issue is resolved (despite having a significantly longer build time).
